vllm/model_executor/layers/layernorm.py

```diff
@@ -20,14 +20,19 @@ def __init__(
         hidden_size: int,
         eps: float = 1e-6,
         var_hidden_size: Optional[int] = None,
+        elementwise_affine: bool = True,
```

Comment:
I suggest naming this something like `has_weight` or `has_gain`. I actually got confused on why it would be named `elementwise_affine` and had to dig a bit (I found [this](https://arxiv.org/pdf/2305.14858) and [this](https://discuss.pytorch.org/t/pytorch-doc-about-layernormalization-is-confusing/26977/4))  -- the thing that confused me here is that this is actually a linear transformation, not just an affine one, since this operation doesn't have a bias that would make it affine.

Reply:
Sure, let's rename it to `has_weight`.